src/js/components/DataSearch/DataSearch.js

```diff
@@ -19,12 +24,15 @@ export const DataSearch = ({ ...rest }) => {
       id={`${dataId}--search`}
       name="_search"
       icon={<Search />}
+      reverse={addSubmit}
```

Comment:
I think I was expecting the search icon to always be at the start, regardless of if the data submits on change or on submit. Having it on the left seems like it aids quick identification of the search field

Reply:
Changed to always be at the start. I think I misinterpreted one of your earlier messages.